### PR TITLE
Add Robust BMA

### PR DESCRIPTION
**Pull request by JorisGoosen**
- https://github.com/jasp-stats/MetaAnalysis/commit/b33e9a2f74ac7587bf6ade5866ad4e1c9423e346
- Implements https://github.com/jasp-stats/INTERNAL-jasp/issues/910

### CODE REVIEW

File: JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R [hunk 1]
```diff
@@ -0,0 +1,1455 @@
+context("Robust Bayesian Meta-Analysis")
+
+# path to the pre-fitted RoBMA model
+fitted_path <- "RoBMA_testfit.RDS"
```
Comment (FBartos) [inline JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: this needs to link to the pre-fitted model.

**Comment by FBartos** [comment]
The unit tests will fail because they need a newer version of the package & set-up the path to the pre-fitted model.

File: JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R (hunk 1 above)
Comment (TimKDJ) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: Is this something Joris should do?

Comment (FBartos) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: anyone who knows how to make it work. The analysis needs to load the .RDS object and I have no idea how are the paths in the unit tests specified.

Comment (JorisGoosen) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: Cant you just do `".\RoBMA_testfit.RDS"` ? It is right next to the unittest R file right?

Comment (FBartos) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: that's how I set it up in the module, but I didn't know whether the location would be the same inside of jasp-desktop

Comment (FBartos) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: and it's loaded from inside the `jasptools::run()` command. I'm just stating that I have no idea how this should be set-up.

Comment (JorisGoosen) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: Me neither, but I figured either @TimKDJ or @vandenman would have a better guess.

Comment (vandenman) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: Something like `file <- file.path(jasptools:::.pkgOptions$tests.dir, "RoBMA_testfit.RDS")` should work if you place the `.RDS` file in the same directory as all the unit tests.

File: Resources/Meta Analysis/description.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
 	"moduleDescription" :
 	{
-		"name": 		"Meta Analysis",
```
Comment (TimKDJ) [inline Resources/Meta Analysis/description.json:4]: Then the folder name in Resources & `RibbonModel` in mainwindow.cpp also need to be changed right?

Schermafbeelding 2020-06-17 om 09 42 47

Comment (JorisGoosen) [reply Resources/Meta Analysis/description.json:4]: Yes, they should, wait, Ill fix this either by reverting or renaming

**Review by TimKDJ — APPROVED** [review]

File: JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R (hunk 1 above)
Comment (FBartos) [reply JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R:4]: it did, thanks!

**Pull request merged**